Detectors/TPC/reconstruction/macro/convertClusterToClusterHardware.C

```diff
@@ -84,6 +95,11 @@ void convertClusterToClusterHardware(TString infile = "o2clus.root", TString out
         oCluster.mSigmaTime2Pre = cluster.getTimeSigma() * cluster.getTimeSigma() * oCluster.mQTot * oCluster.mQTot + oCluster.mTimePre * oCluster.mTimePre;
         oCluster.mRow = cluster.getRow();
         oCluster.mFlags = 0;
+        gsl::span<const MCCompLabel> mcArray = inMCLabels->getLabels(iCurrentCluster + icluster);
+        for (int k = 0;k < mcArray.size();k++) {
+          outMCLabels.addElement(icluster, mcArray[k]);
+          nMCLabels++;
+        }
```

Comment:
would be best to use a range based for loop (or for_each) - AFAIK this makes it easier for compilers to hoist the range check (span is range checked!) out of the loop.